src/core/engines/Cesium/Feature/context.ts

```diff
@@ -1,11 +1,14 @@
 import { createContext, useContext as useReactContext } from "react";
 
+import { LayerEditEvent } from "@reearth/core/Map";
+
 import type { Camera, FlyToDestination, CameraOptions } from "../..";
 
 export type Context = {
   selectionReason?: string;
   getCamera?: () => Camera | undefined;
   flyTo?: (camera: FlyToDestination, options?: CameraOptions) => void;
+  onLayerEdit?: (e: LayerEditEvent) => void;
```

Comment:
Inject `onLayerEdit` prop to the context from props on `engines/Cesium/index.tsx`